### PR TITLE
Enable `--use-cog-base-image` by default

### PR DESCRIPTION
**Pull request by mattt**

### CODE REVIEW

File: pkg/cli/build.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func addDockerfileFlag(cmd *cobra.Command) {
 }
 
 func addUseCogBaseImageFlag(cmd *cobra.Command) {
-	cmd.Flags().BoolVar(&buildUseCogBaseImage, "use-cog-base-image", false, "Use pre-built Cog base image for faster cold boots")
+	cmd.Flags().BoolVar(&buildUseCogBaseImage, "use-cog-base-image", true, "Use pre-built Cog base image for faster cold boots")
```
Comment (andreasjansson) [inline pkg/cli/build.go:113]: Is the correct syntax for disabling this `--use-cog-base-image=false` or `--no-use-cog-base-image`?

**Review by andreasjansson — APPROVED** [review]

**Comment by andreasjansson** [comment]
Base images are still not pulled on all clusters, so shall we hold this off until that's fully deployed?

**Comment by mattt** [comment]
> Base images are still not pulled on all clusters, so shall we hold this off until that's fully deployed?

Merging would only affect anyone running source builds from latest main. So I don't think that's a blocker until we cut a new release.

File: pkg/cli/build.go (hunk 1 above)
Comment (mattt) [reply pkg/cli/build.go:113]: The former. `--no-` prefixes aren't natively supported. See https://github.com/spf13/pflag/issues/214

Comment (mattt) [reply pkg/cli/build.go:113]: That said, we could add a `--no-use-cog-base-image` flag alias

**Comment by andreasjansson** [comment]
Base images have been deployed on most clusters, so I think this is ready to merge.

**Pull request merged**